src/reference/asciidoc/dsl.adoc

```diff
@@ -491,6 +491,34 @@ public IntegrationFlow loggingFlow() {
 }
 ----
 
+[IMPORTANT]
+=====
+The `log()` or `wireTap()` opearators are applied to the current `MessageChannel` (if it is an instane of `ChannelInterceptorAware`) or an intermediate `DirectChannel` is injected into the flow for the currently configured endpoint.
+In the example above the `WireTap` interceptor is added to the `myChannel` directly, because `DirectChannel` implements `ChannelInterceptorAware`:
+[source,java]
+----
+@Bean
+MessageChannel myChannel() {
+    return new DirectChannel();
+}
+
+...
+    .channel(myChannel())
+    .log()
+}
+----
+
+When current `MessageChannel` doesn't implement `ChannelInterceptorAware`, an implicit `DirectChannel` and `BridgeHandler` are injected into the `IntegrationFlow` and only after that a `WireTap` is added to this new `DirectChannel`.
+And when there is no any channel declaration like in this sample:
+[source,java]
+----
+.handle(...)
+.log()
+}
+----
+=====
```

Comment:
You can't put `[source]` within an `IMPORTANT` block - see here https://docs.spring.io/spring-integration/docs/5.0.0.BUILD-SNAPSHOT/reference/html/java-dsl.html#java-dsl-wiretap